server/services/userService.js

```diff
@@ -25,6 +26,7 @@ module.exports = (nomisClientBuilder) => {
       isPrisonUser,
       activeCaseLoad,
       activeCaseLoadId,
+      displayNameInitial: `${forenameToInitial(profile.name)}`,
```

Comment:
Just to check - do we know that a user will always have a name in profile for us to use in forenameToInitial?

Reply:
should do